### PR TITLE
BitmovinAnalyticsCollector (2.0.0-beta1-8-g0388595)

### PR DESCRIPTION
**Pull request by rolandgriesser**

### CODE REVIEW

File: Specs/BitmovinAnalyticsCollector/2.0.0-beta1-8-g0388595/BitmovinAnalyticsCollector.podspec.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      "testspecs": [
+        {
+          "name": "BitmovinPlayerTests",
+          "test_type": "unit",
+          "source_files": "BitmovinAnalyticsCollector/Tests/BitmovinPlayerV3Tests/**/*"
+        }
+      ]
+    }
+  ],
+  "swift_version": "5.0"
```
Comment (stonko1994) [inline Specs/BitmovinAnalyticsCollector/2.0.0-beta1-8-g0388595/BitmovinAnalyticsCollector.podspec.json:71]: This is duplicated 🤔 Already in line 22

File: Specs/BitmovinAnalyticsCollector/2.0.0-beta1-8-g0388595/BitmovinAnalyticsCollector.podspec.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      "source_files": "BitmovinAnalyticsCollector/Classes/BitmovinPlayerV3/**/*.{swift}",
+      "dependencies": {
+        "BitmovinAnalyticsCollector/Core": [
+
+        ]
+      },
+      "tvos": {
+        "dependencies": {
+          "BitmovinPlayer": [
+            "~>3.0.0-rc.6"
```
Comment (stonko1994) [inline Specs/BitmovinAnalyticsCollector/2.0.0-beta1-8-g0388595/BitmovinAnalyticsCollector.podspec.json:51]: `3.0.1` is already out 😉

File: Specs/BitmovinAnalyticsCollector/2.0.0-beta1-8-g0388595/BitmovinAnalyticsCollector.podspec.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        }
+      ]
+    },
+    {
+      "name": "BitmovinPlayer",
+      "source_files": "BitmovinAnalyticsCollector/Classes/BitmovinPlayerV3/**/*.{swift}",
+      "dependencies": {
+        "BitmovinAnalyticsCollector/Core": [
+
+        ]
```
Comment (stonko1994) [inline Specs/BitmovinAnalyticsCollector/2.0.0-beta1-8-g0388595/BitmovinAnalyticsCollector.podspec.json:46]: I'm not 100% familiar with the JSON version of podspecs but AFAIK you don't need the array if you don't specify a version 🤔 (please manually test this)
```suggestion
        "BitmovinAnalyticsCollector/Core"
```

In ruby you would just write:
```
ss.dependency "BitmovinAnalyticsCollector/Core"
```

Comment (rolandgriesser) [reply Specs/BitmovinAnalyticsCollector/2.0.0-beta1-8-g0388595/BitmovinAnalyticsCollector.podspec.json:46]: seems like you need to, I can't remember where I looked that up though.

File: Specs/BitmovinAnalyticsCollector/2.0.0-beta1-8-g0388595/BitmovinAnalyticsCollector.podspec.json (hunk 2 above)
Comment (stonko1994) [reply Specs/BitmovinAnalyticsCollector/2.0.0-beta1-8-g0388595/BitmovinAnalyticsCollector.podspec.json:51]: Makes sense 🙂

**Pull request closed without merging**